src/idiomatic/foundations-api-design/predictable-api/naming-conventions/01-get.md

```diff
@@ -0,0 +1,31 @@
+---
+minutes: 2
+---
+
+# Get
+
+Getting an element from a collection or container.
+
+```rust
```

Comment:
Could we start with a code block that shows the declaration of the method that is the focus of the case study? This would provide a tremendous amount of information about the expected type signature.